Update ediscovery-troubleshooting-common-issues.md (#5532)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md

6. If you still have issues, consider dividing searches that return a large set of results into smaller searches. For example, you can use date ranges in search queries to return a smaller set of results that can be downloaded faster.

> ## Error/issue: Exporting not progressing or stuck

Reviewer: ```suggestion
## Error/issue: Exporting not progressing or is stuck
```

> Exporting search results from eDiscovery or Content Search in the Security and Compliance Center appear stuck or not progressing.

Reviewer: ```suggestion
Exporting search results from eDiscovery or Content Search in the Security and Compliance Center appears stuck or is not progressing.
```

### Resolution

> 1. Re-run the search
> 2. Export the content

Reviewer: ```suggestion
1. Re-run the search.
2. Export the content.
```
